tests/ModelContextProtocol.Tests/Transport/StdioClientTransportTests.cs

```diff
@@ -15,7 +16,7 @@ public async Task CreateAsync_ValidProcessInvalidServer_Throws()
         string id = Guid.NewGuid().ToString("N");
 
         StdioClientTransport transport = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ?
-            new(new() { Command = "cmd", Arguments = ["/C", $"echo \"{id}\" >&2"] }, LoggerFactory) :
```

Comment:
@PederHP @jeffhandley A side effect of this change is that inserting shell pipe syntax in the CLI arguments is no longer possible. I think this is ok given it's a Windows-specific quirk, but I thought I'd double check.

Reply:
Just some ideas, we could: 
* Not change the behavior and defer to users the escaping of characters like &, we could also augment the remarks around `StdioClientTransportOptions.Arguments`.
* Use Arguments instead of ArgumentList, but that requires implementing our own escaping, basically a copy of PasteArguments that wraps the argument in quotes if it contains &.